internal/colorize/colorize.go

```diff
@@ -140,6 +152,10 @@ func colorize(ct ColorTheme, writer io.Writer, arg string) {
 		ct.Disabled(writer)
 	case `ACTIONABLE`:
 		ct.Actionable(writer)
+	case `CAUTION`:
+		ct.Caution(writer)
+	case `ALERT`:
+		ct.Alert(writer)
```

Comment:
It might be worth simply adding things like `[RED]`. So far we've avoided that since we weren't opinionated to that level, but with CVE information it seems appropriate to call out the color.